app/templates/home.html

```diff
@@ -1,14 +1,32 @@
 <div>
-  <h1>Brunch and Booze</h1>
-  <p>Find food and drink pairings for the tastiest brunch ever! Simply search for a food or drink item to see a list of suggested pairings.</p>
+  <h1>Brunch &Booze</h1>
+  <div class="textbox">
+    <p>Find food and drink pairings for the tastiest brunch ever! Simply search for a food or drink item to see a list of suggested pairings.</p>
+  </div>
 </div>
-<div class="food">
-  <i class="fa fa-cutlery fa-5x"></i>
-  <input>
+<div class="search">
+  <div class="images">
+    <i class="fa fa-cutlery fa-5x brunch"></i>
+    <i class="fa fa-glass fa-5x booze"></i>
+  </div>
+  <div class="input">
+    <input type="text" class="brunchinput" ng-model="foodName">
+    <button ng-click="searchBrunch(foodName)"></button>
+    <input type="text" class="boozeinput" ng-model="drinkName">
+    <button ng-click="searchBooze(drinkName)" style="border-color: red"></button>
```

Comment:
This styling is not all permanent! I just used it to tell the difference between the buttons since they appeared next to each other. I will be working on updating the css today.